src/applications/claims-status/containers/YourClaimLetters.jsx

```diff
@@ -38,7 +56,17 @@ export const YourClaimLetters = ({ isLoading, showClaimLetters }) => {
         {lettersLoading ? (
           <va-loading-indicator message="Loading your claim letters..." />
         ) : (
-          <ClaimLetterList letters={letters} />
+          <>
+            <ClaimLetterList letters={currentItems} />
+            {numPages.current > 1 && (
+              <VaPagination
+                onPageSelect={e => onPageChange(e.detail.page)}
+                page={currentPage}
+                pages={numPages.current}
```

Comment:
I think `pages` will be a fixed value. And it needs to be calculated and set on line 29, using `data`

```js
// line 25
const [totalPages, setTotalPages] = useState(0);

// line 29
setTotalPages(Math.round(data.length/ITEMS_PER_PAGE));

// line 65
pages={totalPages}
```

Note: I didn't test this branch locally

Reply:
I can just use the number of chunks in my paginated data to calculate the number of pages, like this:
```
setTotalPages(paginatedItems.length);
```